CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 ### New features
 
+#### Allow creating custom width containers and using them with template
+
+You can now create custom page width container class (this in in addition to already being able to modify the app page width with `$govuk-page-width` variable).
```

Comment:
If we reference the mixin itself will it make clearer that we've introduced new behaviour to an existing mixin?

Reply:
Likewise with the containerClass feature should we be specific what the new feature and how you can use it?

Reply:
Thanks @nickcolley. I've updated this section, have a look now.